src/modules/bunnymq.js

```diff
@@ -49,10 +49,14 @@ module.exports = (connection) => {
     instance.connection = connection;
   }
   return {
-    consume: instance.consume.bind(instance),
-    subscribe: instance.subscribe.bind(instance),
-    produce: instance.produce.bind(instance),
-    publish: instance.publish.bind(instance),
-    connection: instance.connection
+    connection: instance.connection,
+    consumer: {
+      consume: instance.consume.bind(instance),
+      subscribe: instance.subscribe.bind(instance)
+    },
+    producer: {
+      produce: instance.produce.bind(instance),
+      publish: instance.publish.bind(instance)
```

Comment:
The only thing is that we need to keep ``procuder`` and ``consumer`` for ``produce`` and ``consume`` functions only (just to keep the backwards compat). ``subscribe`` and ``publish`` should be exposed out of the object as requested [here](https://github.com/dial-once/node-bunnymq/issues/41) @dial-once/developers Do you think we need to do it? Or keep it as Yacine has it right now?